### DISCUSSION TITLE
您好 已经买了Apple tv 求邀请码

### DISCUSSION THREAD

**Discussion by bryancc2012** [Paladinfeng/MiaoProject#124] [General]
您好 已经买了Apple tv 求邀请码

**Comment by Paladinfeng** [comment]
测试已经满载

**Reply by bryancc2012** [reply]
感谢大神回复。唉，啥都齐了还是看不了B站的杜比视界呀，SONY A90J, apple TV... 

什么时候会有更多名额放出么？bryancc2012@gmail.com

**Reply by bryancc2012** [reply]
每天上来试一下，有空出来的邀请码么？

**Reply by bryancc2012** [reply]
什么时候会扩容么？

**Reply by Paladinfeng** [reply]
TF版本已经达到人数总上限，不可能扩容了，如果真的很需要，就商店版吧
